features/exacte_match.feature

```diff
@@ -0,0 +1,188 @@
+  | 3581KL   | 49         |            | A                    | 0344200000172058              |
+
+Scenario: Zoeken met combinatie postcode en huisnummer
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 2391PH   | 1          |            |                      | false       |
+  Dan bevat het resultaat de volgende objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 2391PH   | 1          |            |                      |
+  | 2391PH   | 1          | A          |                      |
+  | 2391PH   | 1          | A          |  1                   |
+  | 2391PH   | 1          | A          |  2                   |
+  | 2391PH   | 1          | A          |  3                   |
+  | 2391PH   | 1          | A          |  4                   |
+  | 2391PH   | 1          | A          |  5                   |
+  | 2391PH   | 1          | A          |  6                   |
+  | 2391PH   | 1          | A          |  7                   |
+  | 2391PH   | 1          | A          |  8                   |
+  | 2391PH   | 1          | B          |                      |
+  | 2391PH   | 1          | C          |                      |
+  | 2391PH   | 1          | D          |                      |
+  | 2391PH   | 1          | E          |                      |
+  | 2391PH   | 1          | F          |                      |
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 2391PH   | 1          |            |                      | true        |
+  Dan bevat het resultaat de volgende objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 2391PH   | 1          |            |                      |
+
+Scenario: zoeken met combinatie postcode, huisnummer en huisnummertoevoeging
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 2594AN   | 265        |            | 2                    | false       |
+  Dan bevat het resultaat de volgende objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 2594AN   | 265        | A          | 2                    |
+  | 2594AN   | 265        | B          | 2                    |
+  | 2594AN   | 265        | C          | 2                    |
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 2594AN   | 265        |            | 2                    | true        |
+  Dan bevat het resultaat geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
```

Comment:
Deze tabel kan weg. In de Dan wordt al aangegeven dat het resultaat geen objecten bevat